scripts/microbe_stage/organelle_table.as

```diff
@@ -553,7 +553,7 @@ void setupOrganelles(){
 
     _addOrganelleToTable(Organelle(respiratoryProtein));
 
-    // chromatophors
+    // chromatophors, now called thakyloids in the gui
```

Comment:
I think there's a typo here.

Reply:
Yeah I agree, should be Thylakoids. Are you ok with editing it or shall I do it, I'd need to boot into linux which is a bit of a hassle but doable.

Reply:
I'll fix it then.